lib/wda/webdriveragent.js

```diff
@@ -275,29 +275,38 @@ class WebDriverAgent {
    */
   async setupCaching (updatedWDABundleId) {
     const status = await this.getStatus();
-    let currentAppBundleID;
     if (!status || !status.build) {
-      log.debug(`WDA is currently not running`);
+      log.debug('WDA is currently not running. There is nothing to cache');
       return;
     }
 
-    if (util.hasValue(status.build.productBundleIdentifier)) {
-      currentAppBundleID = status.build.productBundleIdentifier;
+    const {
+      productBundleIdentifier,
+      upgraded_at,
```

Comment:
This should be translated into camel case for consistency.

Reply:
https://github.com/appium/WebDriverAgent/pull/112